### PR TITLE
Task 1.1-1.5 | HTML, CSS & Git Basics

### PR DESCRIPTION
**Pull request by foxtrot2609**
This is my personal CV based on HTML5 and CSS3 with flex-box and media query.

Could you give me some advices and check my CV before merge?

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
+body {
+  box-sizing: border-box;
+  max-width: 1024px;
+  min-width: 480px;
+  margin: 0 auto;
+  border: 30px solid #aa4b9d;
```
Comment (TatyanaKavalenkaTR) [inline style.css]: Let's exclude so aggressive magenta color, to not scare anyone :)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      </section>
+    </div>
+    <div class="my-photo">
+      <img src="/isachanka.jpg" alt="my photo">
+    </div>
+  </header>
+  <main>
+    <div class="left-col">
+      <section>
+        <h2>Summary</h2>
```
Comment (TatyanaKavalenkaTR) [inline index.html]: Let's make the summary full-width as it is in example (https://drive.google.com/file/d/1GOT8bp6g_UAJ5iOX3orj1W0NDa5wDwvw/view) . It is hard to read such long text sized in one column.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  font-family: Verdana, sans-serif;
+}
+
+header {
+  display: flex;
+  font-family: Arial, sans-serif;
```
Comment (TatyanaKavalenkaTR) [inline style.css]: As a best practice, we use the same font across all application.

File: style.css (hunk 1 above)
Comment (foxtrot2609) [reply style.css]: Thanks for advise. I have changed color.

File: index.html (hunk 2 above)
Comment (foxtrot2609) [reply index.html]: Agree with you. I have changed structure of body.

File: style.css (hunk 3 above)
Comment (foxtrot2609) [reply style.css]: Thanks. I have used one font for all application.

**Pull request merged**